### ISSUE TITLE
Improve reliability of high batch size training

### ISSUE THREAD

**Issue by armandmcqueen** [armandmcqueen/tensorpack-mask-rcnn#53]
Currently 32 GPUs with a per-gpu-batch-size of 4 (32x4) will sometimes converge to target accuracy and other times reach a NaN total loss. We need to eliminate the occurrence of NaN.

**Comment by armandmcqueen** [comment]
Tensorpack Gradient Clipping - https://github.com/tensorpack/tensorpack/issues/1131

**Comment by armandmcqueen** [comment]
With corrected learning rate, we no longer get NaNs for 32 GPUs.
